### PR TITLE
chore: remove paths from tsconfig

### PR DESCRIPTION
**Pull request by Tambouil**
### 📚 Description

Paths are no longer needed in TSconfig since last TS update

### CODE REVIEW

**Comment by Julien-R44** [comment]
thanks for the 3 PRs dude !

**Comment by Julien-R44** [comment]
Oh, just one thing

we should also update the version of typescript in these PRs. Pin typescript 5.4 because that's the version that added this feature

**Comment by Tambouil** [comment]
oh yes you're right ! thanks for the feedback

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "pino-pretty": "^10.3.1",
     "prettier": "^3.2.4",
     "ts-node": "^10.9.2",
-    "typescript": "^5.3.3"
+    "typescript": "^5.4.3",
```
Comment (Julien-R44) [inline package.json]: trailing comma is not allowed in json files

Comment (Tambouil) [reply package.json]: my bad, I fixed it

**Comment by Julien-R44** [comment]
Thanks!

**Pull request merged**